opre_ops/opre_ops/settings/static/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore everything in this directory, added to allow collectstatic to put
+# static files someplace
+!.gitignore
```

Comment:
I created `settings/static` because of advice from [the cg-support slack channel](https://gsa-tts.slack.com/archives/C09CR1Q9Z/p1648220372530379?thread_ts=1648160052.184139&cid=C09CR1Q9Z), and the `.gitignore` to allow this otherwise-empty directory to be in source control. But, if it doesn't seem right to you, I'll remove it.